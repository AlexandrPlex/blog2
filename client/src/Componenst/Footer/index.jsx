// @flow

import React from 'react';
import './footerStyle.scss';

export default class Footer extends React.PureComponent<{}> {
    render () {
        return <div className='headerStyle'>
            <h1 className='logo'>Footer</h1>
        </div>;
    }
}
