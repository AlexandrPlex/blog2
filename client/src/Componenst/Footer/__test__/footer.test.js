import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../index';

it('Footer Snaphot', () => {
    const tree = renderer
        .create(<Footer/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
