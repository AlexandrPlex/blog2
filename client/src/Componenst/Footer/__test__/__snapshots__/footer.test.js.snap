// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Snaphot 1`] = `
<div
  className="headerStyle"
>
  <h1
    className="logo"
  >
    Footer
  </h1>
</div>
`;
